fineract-provider/src/main/java/org/apache/fineract/portfolio/self/loanaccount/api/SelfLoansApiResource.java

```diff
@@ -171,8 +175,11 @@ public String template(@QueryParam("clientId") final Long clientId,
     @POST
```

Comment:
Incomplete

Reply:
What is incomplete here?

Reply:
all post methods are not mentioned!
refer to the current documentation

Reply:
Yes, Thats what I have mentioned in the description, only documentation of one POST endpoint is there so I have added for only one